app/main.js

```diff
@@ -942,27 +955,37 @@ function postponeBreak (shouldPlaySound = false) {
   updateTray()
 }
 
-function skipToMicrobreak () {
+function skipToMicrobreak (delay = 0) {
```

Comment:
No default value needed. (If we add it as `skipToMicrobreak (delay = 100)` in `BreaksPlanner`)

Reply:
Indeed. Actually, this was superfluous even without that. Fixed.